run_tools/law_customizations.py

```diff
@@ -92,7 +93,7 @@ def load_sample_configs(self):
         self.samples = { key : samples[key] for key in selected_samples }
 
     def store_parts(self):
-        return (self.__class__.__name__, self.version)
+        return (self.__class__.__name__, self.version, self.period)
```

Comment:
```suggestion
        return (self.__class__.__name__, self.period, self.version)
```
to be compatible with `central_anaTuples_path`

Reply:
ok